src/dlgTriggerEditor.cpp

```diff
@@ -520,11 +519,22 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
     connect(mProfileSaveAction, &QAction::triggered, this, &dlgTriggerEditor::slot_profileSaveAction);
 
     mProfileSaveAsAction = new QAction(QIcon(qsl(":/icons/utilities-file-archiver.png")), tr("Save Profile As"), this);
-    mProfileSaveAsAction->setEnabled(true);
     connect(mProfileSaveAsAction, &QAction::triggered, this, &dlgTriggerEditor::slot_profileSaveAsAction);
 
-    auto *nextSectionShortcut = new QShortcut(QKeySequence(Qt::CTRL | Qt::Key_Tab), this);
-    connect(nextSectionShortcut, &QShortcut::activated, this, &dlgTriggerEditor::slot_nextSection);
+    if (mpHost->loadedOk) {
+        mProfileSaveAction->setEnabled(true);
+        mProfileSaveAsAction->setEnabled(true);
+    } else {
```

Comment:
The default is for a `QAction` to be created in the enabled state...
```suggestion
    if (!mpHost->loadedOk) {
```

Reply:
However if you do want to retain a `if (mpHost->loadedOk)` (and why not call it `mLoadedOk`!) branch, you can put the two `QObject::connect(...)` calls for the two save actions into it...

Reply:
Thanks, applied this.